templates/zerver/compose.html

```diff
@@ -78,6 +79,9 @@
                      id="attach_files" href="#" title="{{ _('Attach files') }}"></a>
                   <a class="message-control-button icon-vector-font"
                      href="#markdown-help" title="Formatting" data-toggle="modal"></a>
+                  <a class="preview_text" style="display:none;"
```

Comment:
@showell Changed the name to `markdown_preview_button` as we have to replace the `text` with a suitable icon in the near future.